vm/datemath.go

```diff
@@ -187,7 +187,8 @@ func (d *DateConverter) findDateMath(node expr.Node) {
 			d.findDateMath(arg)
 		}
 	case *expr.IncludeNode:
-		if err := resolveInclude(d.ctx, n, 0); err != nil {
+
+		if err := resolveInclude(d.ctx, n, 0, make([]string, 0)); err != nil {
```

Comment:
Normally I would say a `map[]` would be better for this because a `slice` is actually copied at each method invocation, so this slice is going to be copied quite a bit.

But in this case the slice is probably the way to go, since we are trying to make an ordered path to include in the error, and it is probably a shallow cycle that is causing this type of error.

We will see how the performance is with the slice...